honeybee_3dm/tojson.py

```diff
@@ -0,0 +1,39 @@
+try:  # import the core honeybee dependencies
+    from honeybee.model import Model
+    from honeybee.config import folders
+except ImportError as e:
+    raise ImportError('\nFailed to import honeybee:\n\t{}'.format(e))
+
+import os
+import json
+
+
+def publish_json(_hb_objs, _name_=None, _folder_=None, indent_=None, abridged_=False, _dump=True):
```

Comment:
I don't think that we use the term `publish` in our code-base at all. It should probably change to `to_json`. @chriswmackey is the right person to confirm. If he is fine with publish then ignore this comment.

Reply:
Revised.